### ISSUE TITLE
[ICON REQUEST]: <Motoko>

### ISSUE THREAD

**Issue by Kellsonphilips** [devicons/devicon#1334]
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Motoko is a programming language for building directly on the internet for DApps.

### Links and sources

[Motoko Programming Language](https://internetcomputer.org/docs/current/developer-docs/build/languages/motoko/)
[Motoko Dfinity](https://stackoverflow.blog/2020/08/24/motoko-the-language-that-turns-the-web-into-a-computer/)
[Motoko Forum](https://forum.dfinity.org)

### Additional information

The Motoko programming language is a new, modern and type safe language for developers who want to build the next generation of distributed applications to run on the Internet Computer blockchain network. Motoko is specifically designed to support the unique features of the Internet Computer and to provide a familiar yet robust programming environment. As a new language, Motoko is constantly evolving with support for new features and other improvements.

The Motoko compiler, documentation and other tooling is [open source](https://github.com/dfinity/motoko) and released under the Apache 2.0 license. Contributions are welcome.

![motoko-pic](https://user-images.githubusercontent.com/81332784/181936085-266a58d6-5ebc-45b3-9e3a-830871582420.png)

![Motoko](https://user-images.githubusercontent.com/81332784/181936110-419f7ad2-460c-4d61-a7eb-2c4af9ee3bf6.jpeg)

**Comment by BenSouchet** [comment]
Hi, the project seems to be at an early stage, I cannot find a place (Forum, Github repositories, ...) where they use the mascot as logo / icon.

In my opinion it would be good to wait a bit before adding it to the project, or ask officially on their forum what is the logo, and do they have an SVG version available?

**Comment by Snailedlt** [comment]
I agree with @BenSouchet, before we have some concrete evidence or statement saying this is their official logo, I'm hesitant to implement it

**Comment by Kellsonphilips** [comment]
I will have to get verification from the forum if that's their official icon and if they have SVG then refer back to @BenSouchet comment.

**Comment by Snailedlt** [comment]
@Kellsonphilips Any updates?

**Comment by domwoe** [comment]
The official logo(s) can be found at https://dfinity.frontify.com/d/XzkdhhDptijE/dfinity-brand-guide#/motoko/logo
On the top right, there's a link to download all logos. This includes an SVG version.

**Comment by Kellsonphilips** [comment]
> The official logo(s) can be found at https://dfinity.frontify.com/d/XzkdhhDptijE/dfinity-brand-guide#/motoko/logo
> On the top right, there's a link to download all logos. This includes an SVG version.

Thanks, @domwoe  for helping with the clarification and providing a src
@Snailedlt check that out and hope it's what you are looking for.

**Comment by Snailedlt** [comment]
@Kellsonphilips ohh they're out now. Neat! Thnaks for linking the logos 💯

**Comment by Kellsonphilips** [comment]
Cool

**Comment by Kellsonphilips** [comment]
@Snailedlt issue updated
